errbot/config-template.py

```diff
@@ -93,6 +93,12 @@
 # For example CORE_PLUGINS = ('ACLs', 'Backup', 'Help') you get those names from the .plug files Name entry.
 # For absolutely no plug: CORE_PLUGINS = ()
 
+# Defines an order in which the plugins are getting their callbacks. Useful if you want to have plugins do
+# pre- or post-processing on messages.
+# The 'None' tuple entry represents all the plugins that aren't to be expicitely ordered. For example, if
```

Comment:
If the other plugins order doesn't matter, you don't have to specify where they are placed. If they does matter then you'll have to specify it here. It makes the behavior clearer and no need to add the weird None parameter.